src/main/java/uk/gov/hmcts/reform/iacaseapi/infrastructure/controllers/PreSubmitCallbackController.java

```diff
@@ -145,12 +145,23 @@ public PreSubmitCallbackController(
         PreSubmitCallbackResponse<AsylumCase> callbackResponse =
             callbackDispatcher.handle(callbackStage, callback);
 
-        log.info(
-            "Asylum Case CCD `{}` event `{}` handled for Case ID `{}`",
-            callbackStage,
-            callback.getEvent(),
-            callback.getCaseDetails().getId()
-        );
+        if (!callbackResponse.getErrors().isEmpty()) {
```

Comment:
Could possibly flip this if/else around

Reply:
Thanks @bradsorour . will do it along with any future changes.